src/components/BrowserFilter/BrowserFilter.react.js

```
@@ -5,35 +5,42 @@
+import Checkbox from 'components/Checkbox/Checkbox.react';
+import Field from 'components/Field/Field.react';
 import Filter from 'components/Filter/Filter.react';
-import FilterRow from 'components/BrowserFilter/FilterRow.react';
 import Icon from 'components/Icon/Icon.react';
+import Label from 'components/Label/Label.react';
 import Popover from 'components/Popover/Popover.react';
-import Field from 'components/Field/Field.react';
 import TextInput from 'components/TextInput/TextInput.react';
-import Label from 'components/Label/Label.react';
+import { CurrentApp } from 'context/currentApp';
+import { List, Map } from 'immutable';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)